prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-toa-maintenance-view-page/enrollee-toa-maintenance-view-page.component.html

```diff
@@ -0,0 +1,21 @@
+<app-page [busy]="busy"
+          showToolbar="true"
+          (back)="onBack()">
+  <app-page-header>TOA Maintenance</app-page-header>
+
+  <app-page-subheader *ngIf="enrolleeAgreementVersion"
+                      class="mb-4"
+                      [divider]="false">
+    <ng-container appPageSubheaderTitle>
+      {{ AgreementType[enrolleeAgreementVersion.agreementType] | replace: '_':' ' | capitalize: true }}
+    </ng-container>
+  </app-page-subheader>
+
+  <app-page-subheader>
+    <ng-container appPageSubheaderTitle>
+      Preview
+    </ng-container>
+  </app-page-subheader>
+
+  <div [innerHtml]="enrolleeAgreementVersion?.text"></div>
```

Comment:
Note that the TOA template HTML has custom CSS class, which is in `@import 'mixins/agreements';` The final rendering style is slightly different if the css is not imported into `enrollee-toa-maintenance-view-page.component.scss`